companion/src/modeledit/expodialog.cpp

```diff
@@ -240,7 +240,7 @@ void ExpoDialog::fmInvertAll()
 {
   lock = true;
   for (int i=0; i<CPN_MAX_FLIGHT_MODES; i++) {
-    cb_fp[i]->setChecked(not cb_fp[i]->checkState());
+    cb_fp[i]->setChecked(!cb_fp[i]->checkState());
```

Comment:
I understand, and yes, please change it since you're "in there" already.  I'm not sure if @elecpower is planning any further changes after my [previous comments](https://github.com/opentx/opentx/pull/4494#issuecomment-282627072) or not.